actionpack/lib/action_controller/metal/cookies.rb

```diff
@@ -6,9 +6,8 @@ module Cookies
       helper_method :cookies if defined?(helper_method)
     end
 
-    private
```

Comment:
Don't see any way to avoid making this public. I need to access cookies from within the strategy class.